plerkle_snapshot/README.md

```diff
@@ -1,5 +1,91 @@
-This bin contains alot of code lifted from https://github.com/terorie/solana-snapshot-etl/
+# Solana Snapshot ETL 📸
 
-terorie is an inspiration to the masses and I hope to be as good of a dev as this anon someday.
+[![crates.io](https://img.shields.io/crates/v/solana-snapshot-etl?style=flat-square&logo=rust&color=blue)](https://crates.io/crates/solana-snapshot-etl)
+[![docs.rs](https://img.shields.io/badge/docs.rs-solana--snapshot--etl-blue?style=flat-square&logo=docs.rs)](https://docs.rs/solana-snapshot-etl)
+[![license](https://img.shields.io/badge/license-Apache--2.0-blue?style=flat-square)](#license)
 
-IGNORE ME FOR NOW as we need to fix something in the underlying library from terrorie to have slot
\ No newline at end of file
+**`solana-snapshot-etl` efficiently extracts all accounts in a snapshot** to load them into an external system.
+
+> [!IMPORTANT]  
+> This code is a fork of the [original repository](https://github.com/riptl/solana-snapshot-etl.git) and has been modified to diverge from the behavior of the original implementation.
```

Comment:
I did a high-level glance at this code in the original repository and compared it to what is being added in this PR.  It looked about the same to me except looked like some options such as dumping to SQLite3 were removed.

Could you provide here a high level indication of what has been modified from the original implementation?